### PR TITLE
Add support for the pull request review thread event

### LINKED ISSUE #2324: Webhook event type `pull_request_review_thread` missing
It seems like GitHub is sending the webhook event type `pull_request_review_thread` when someone resolves a comment in a PR.

Webhook event schema can be seen here: https://github.com/octokit/webhooks/blob/d93eebf4c48dc3c36f9c7ceffab356fa444bf9fc/payload-schemas/api.github.com/pull_request_review_thread/resolved.schema.json

The event doesn't seem to be documented in [Webhook events and payload](https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#pull_request_review_thread) though.

### PR DESCRIPTION
**Pull request by taisph**
Fixes #2324

### CODE REVIEW

File: github/pulls_threads.go
Comment (gmlewis) [inline github/pulls_threads.go:8]: Please continue. :smile:

**Review by gmlewis — CHANGES_REQUESTED** [review]

**Comment by taisph** [comment]
I added the link https://docs.github.com/en/developers/webhooks-and-events/webhook-events-and-payloads#pull_request_review_thread which presumably will be the correct link once they update the docs by looking at the existing structure. I don't know what to expect with regards to when they'll update it as I managed to find [a misnamed issue](https://github.com/github/docs/issues/12295) about the missing documentation that was created back in November 2021.

 Should this PR be kept back until the docs are updated?

**Comment by gmlewis** [comment]
> Should this PR be kept back until the docs are updated?

I don't think that's necessary.  Many endpoints have been added in the based just based on observing the response received, so I think this is fine. It can always be updated once the official docs are released. Thank you, @taisph !

**Review by gmlewis — APPROVED** [review]
Thank you, @taisph !
LGTM.

Awaiting second LGTM+Approval from any other contributor to this repo before merging.

**Review by raynigon — APPROVED** [review]
LGTM

**Comment by gmlewis** [comment]
Thank you, @raynigon !
Merging.

**Pull request merged**
